src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props.in

```diff
@@ -1,5 +1,10 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-	<!-- Android Sdk Tool versions are sourced from this file. This is maintained in the android-tools repo. -->
+	<PropertyGroup>
+		<AndroidSdkBuildToolsVersion Condition=" '$(AndroidSdkBuildToolsVersion)' == '' ">@SDK_BUILD_TOOLS_VERSION@</AndroidSdkBuildToolsVersion>
+		<AndroidSdkPlatformVersion Condition=" '$(AndroidSdkPlatformVersion)' == '' ">@SDK_PLATFORM_VERSION@</AndroidSdkPlatformVersion>
+		<AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">@SDK_PLATFORM_TOOLS_VERSION@</AndroidSdkPlatformToolsVersion>
+	</PropertyGroup>
+	<!-- Android Sdk Tool versions are sourced from this file. This is maintained in the dotnet/android-tools repo. -->
 	<Import Project="$(MSBuildThisFileDirectory)\Xamarin.Android.Tools.Versions.props" />
```

Comment:
I think part of this goal would be to eventually remove `Xamarin.Android.Tools.Versions.props` from dotnet/android-tools? I think it would make sense to remove this import now, and update the properties above to also include the other properties in that file:

```xml
    <AndroidCommandLineToolsVersion>
    <AndroidSdkEmulatorVersion>
    <AndroidNdkVersion>
    <JavaSdkVersion>
```

We could base these off other values already defined in Configuration.props as well, to continue to improve our own build and test coverage against the versions of these tools that we're recommending:

https://github.com/dotnet/android/blob/64bb147f2ead5bd403a823d27b33f590197477fa/Configuration.props#L162
https://github.com/dotnet/android/blob/64bb147f2ead5bd403a823d27b33f590197477fa/Configuration.props#L166

NDK and JDK versions may be a bit more tricky as we'd potentially need to introduce new props for these and also pass them to xaprepare:

https://github.com/dotnet/android/blob/05bc90a09e1bf51f467355db2708e062751b0f5c/build-tools/xaprepare/xaprepare/ConfigAndData/BuildAndroidPlatforms.cs#L9

https://github.com/dotnet/android/blob/553e9274dcef760b6305dbd8b130e9136c60be68/build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.cs#L21